content/sensu-go/5.13/reference/rbac.md

```diff
@@ -808,6 +808,7 @@ description  | Username or group name.
 required     | true 
 type         | String
 example      | {{< highlight shell >}}"name": "alice"{{< /highlight >}}
+example with prefix | {{< highlight shell >}}"name": "ladp:alice"{{< /highlight >}}
```

Comment:
```suggestion
example with prefix | {{< highlight shell >}}"name": "ad:alice"{{< /highlight >}}
```

Reply:
Just suggest changing to `ad` to be consistent with the group prefix examples you give later.